h1st/model/fuzzy/fuzzy_variables.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from typing import Any
+
+import skfuzzy
+from skfuzzy import control as skctrl
+
+from h1st.model.fuzzy.enums import FuzzyMembership as fm
```

Comment:
OK, so `FuzzyMembership` is only needed to be referenced in `FuzzyVariables`. Makes sense.